repository/spark/operator/templates/spark-operator-deployment.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Name }}
     app.kubernetes.io/version: {{ .Params.operatorVersion }}
 spec:
-  replicas: 1
+  replicas: {{ .Params.replicas }}
```

Comment:
it makes sense to wrap this property in `{{- if eq .Params.enableLeaderElection "true" }}` condition to avoid the installation of multiple controllers without leader election.

Reply:
Yes, sounds good, made it conditional on leader election switch